docs/platforms/apple/common/user-feedback/index.mdx

```diff
@@ -1,15 +1,113 @@
+
+### Customization
+
+You can customize the widget by setting various properties in the `SentryUserFeedbackWidgetConfiguration` and `SentryUserFeedbackFormConfiguration` classes:
+
+```swift
+let widgetConfig = SentryUserFeedbackWidgetConfiguration()
+widgetConfig.autoInject = true
+widgetConfig.triggerLabel = "Feedback"
+widgetConfig.triggerAccessibilityLabel = "Submit Feedback"
+
+// Form customization
+let formConfig = SentryUserFeedbackFormConfiguration()
+formConfig.showName = true
+formConfig.isNameRequired = false
+formConfig.enableScreenshot = true
+formConfig.formTitle = "Report an Issue"
+formConfig.submitButtonLabel = "Submit"
+formConfig.cancelButtonLabel = "Cancel"
+
+let feedbackConfig = SentryUserFeedbackConfiguration { config in
+    config.widgetConfig = widgetConfig
+    config.uiFormConfig = formConfig
+}
+
+let feedbackIntegration = SentryUserFeedbackIntegration(configuration: feedbackConfig)
+feedbackIntegration.createWidget()
+```
+
+This configuration customizes the widget's button text, form fields, and whether the user can attach screenshots.
+
+### Session Replay
+
+The User Feedback widget integrates seamlessly with Session Replay. When the widget is opened, the Replay SDK buffers up to 30 seconds of the user's session. If feedback is submitted, this replay is sent along with the feedback, allowing you to view both the feedback and the user's actions leading up to the feedback submission.
+
+### User Feedback API
+
+The User Feedback API allows you to collect user feedback while using your own UI components. You can submit feedback directly using the `captureFeedback` method in the `SentryUserFeedbackIntegration` class:
+
+```swift
+feedbackIntegration.captureFeedback(
+    message: "I encountered a bug while using the app.",
+    name: "John Doe",
+    email: "john.doe@example.com",
+    hints: ["page": "Settings"]
+)
+```
+
+This method associates the feedback with the corresponding event in Sentry, giving you valuable insights into user issues.
+
+Alternatively, you can use the [User Feedback API endpoint](/api/projects/submit-user-feedback/) directly if you prefer to handle the HTTP request manually.
+
+## Crash-Report Modal
+
+The Cocoa-based Crash-Report Modal is a customizable feature that prompts users for feedback when an error occurs. This modal captures the user's name, email, and a description of the issue, pairing the feedback with the event in Sentry.
+
+### Integration
+
+To integrate the Crash-Report Modal in your Cocoa app, configure it in your Sentry initialization:
+
+```swift
+SentrySDK.showReportDialog(eventId: eventId) {
+    print("Crash report dialog shown")
+}
+```
+
+This will display a modal dialog where users can provide feedback related to a specific error event.
+
+---
+
+This adapted documentation aligns with the changes and functionality we implemented in the Cocoa platform, providing clear instructions for setting up and customizing the User Feedback widget in a Cocoa application.
```

Comment:
```suggestion
```

this seems unnecessary to include? It might also make more sense in the beginning instead of the end of the page to give the user context

Reply:
Yep, this wasn't supposed to be included.